Automattic/wp-calypso
# Stats: Add arrow navigation to the Posting Activity section

#### Proposed Changes

* Add arrow navigation so users can scan the data from previous years
* Remove the `moment.js` usage from the `PostTrends` component
* Turn the `PostTrends` component from a Class component to a Function component

#### Testing Instructions

* Go to the Calypso live link
* Navigate to the Insights tab
* Ensure the current site has data available for the Posting Activity section
* On the top right of the Posting Activity section should appear two arrow buttons that you can use to see the data from different years

#### Screenshots

![image](https://user-images.githubusercontent.com/6406071/210114279-2ff4360a-7816-4012-84eb-7a7d37b70bba.png)
![image](https://user-images.githubusercontent.com/6406071/210114294-9b6d3cb9-bd24-48fe-88a8-427f9642d5ed.png)



#### Pre-merge Checklist





Related to #70609
